tools/c7n_azure/c7n_azure/resources/route_table.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Capital One Services, LLC
```

Comment:
Please add new resource to the docs with some examples.

Reply:
will do

Reply:
I just committed the docs. I am not too familiar with sphinx & .rst, so review on that would be good!